### DISCUSSION TITLE
Custom Api Integration

### DISCUSSION THREAD

**Discussion by akhtaralikhandev** [danny-avila/LibreChat#1997] [Troubleshooting]
I have my own api . I want to send questions to over there and get back the response

**Comment by fuegovic** [comment]
You can add custom endpoints that follow the OpenAI format with the librechat.yaml config file

see: https://docs.librechat.ai/install/configuration/custom_config.html

**Reply by akhtaralikhandev** [reply]
let's say I have my model somewhere and I want to send api request like for example . https://api5.abcchat/abc . will this work ? plz guide me thanks
